src/client/app/sharedComponents/searchWithDropdown/SelectedValues.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { ChevronDownIcon, ChevronUpIcon, TrashIcon } from '@navikt/aksel-icons';
+import { Button, Chips } from '@navikt/ds-react';
+
+type groupObject = {
+	label: string;
+	value: string;
+	group?: string;
+};
+
+interface Props {
+	values: groupObject[];
+	remove: (value: string) => void;
+	removeAllValues: () => void;
+}
+
+const Group = ({ group, chips, remove }: { group: string; chips: Props['values']; remove: Props['remove'] }) => {
+	const [open, setOpen] = useState(false);
+
+	return (
+		<div className={open ? `bg-white rounded-md p-1.5 pt-0 pl-0 w-full` : ''}>
+			<Button
+				style={{ padding: '4px' }}
```

Comment:
Fungerer det ikke med `className="p-1"`?

Reply:
jo, men ble overskrevet av annen css. Men kunne visst bruke className="!p-1" for å sette important. Det visste jeg ikke 👍